test/functional/test-ad-cid.js

```diff
@@ -73,7 +73,7 @@ function tests(name) {
         });
 
         it('provides cid to ad', () => {
-          adConfig['_ping_'] = {clientIdScope: cidScope};
+          config.clientIdScope = cidScope;
```

Comment:
I'm late to this conversation, I know.  But it looks to me like `adConfig['__ping__']` is not getting reset between tests.  All of these modifiers are just applying to it sequentially.  Which works, but makes the tests potentially order-dependent.  Am I missing something?